### ISSUE TITLE
JWT config is not working with minikube on macos

### ISSUE THREAD

**Issue by ahmed-adly-khalil** [Kong/kubernetes-ingress-controller#920]
Hello,
I have tried to follow the guide here https://github.com/Kong/kubernetes-ingress-controller/blob/main/docs/guides/configure-acl-plugin.md and after applying creating the plugin and applying it to the ingress, still the route gives 200 response not 401. Here is the detailed steps:

```
 minikube delete
 minikube start # starting fresh

 kubectl create -f https://bit.ly/k4k8s

export PROXY_IP=$(minikube service -n kong kong-proxy --url | head -1)

echo $PROXY_IP # result --> http://192.168.64.16:30489

curl -i $PROXY_IP # result --> 404

kubectl apply -f https://bit.ly/k8s-httpbin 
# result -> 
# service/httpbin created
# deployment.apps/httpbin created


echo '
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: demo-get
  annotations:
    konghq.com/strip-path: "false"
    kubernetes.io/ingress.class: kong
spec:
  rules:
  - http:
      paths:
      - path: /get
        backend:
          serviceName: httpbin
          servicePort: 80
' | kubectl apply -f -
# result --> ingress.extensions/demo-get created

echo '
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: demo-post
  annotations:
    konghq.com/strip-path: "false"
    kubernetes.io/ingress.class: kong
spec:
  rules:
  - http:
      paths:
      - path: /post
        backend:
          serviceName: httpbin
          servicePort: 80
' | kubectl apply -f -

# result --> ingress.extensions/demo-post created

curl -i $PROXY_IP/get # result --> 200

 curl -i --data "foo=bar" -X POST $PROXY_IP/post  # result --> 200


echo "
apiVersion: configuration.konghq.com/v1
kind: KongPlugin
metadata:
  name: app-jwt
plugin: jwt
" | kubectl apply -f -

# result --> kongplugin.configuration.konghq.com/app-jwt created


echo '
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: demo-get
  annotations:
    plugins.konghq.com: app-jwt
    konghq.com/strip-path: "false"
    kubernetes.io/ingress.class: kong
spec:
  rules:
  - http:
      paths:
      - path: /get
        backend:
          serviceName: httpbin
          servicePort: 80
' | kubectl apply -f -

# result --> ingress.extensions/demo-get configured


echo '
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: demo-post
  annotations:
    plugins.konghq.com: app-jwt
    konghq.com/strip-path: "false"
    kubernetes.io/ingress.class: kong
spec:
  rules:
  - http:
      paths:
      - path: /post
        backend:
          serviceName: httpbin
          servicePort: 80
' | kubectl apply -f -

# result --> ingress.extensions/demo-post configured

curl -i $PROXY_IP/get 

###################
# result --> 200 and it should be 401
###################

````

any clue why this is happening? i followed the docs to the T 

Thank you,

**Comment by ahmed-adly-khalil** [comment]
i found the issue, 

documentation example has the incorrect annotation on the ingress rule for the plugin, it should be
`konghq.com/plugins:  app-jwt`
instead of `plugins.konghq.com: app-jwt`

Thanks to @Satwant on K8s slack 

Please update the docs

**Comment by ahmed-adly-khalil** [comment]
The first issue is resolved as i mentioned. However, after following the entire guide, i'm still getting  `{"message":"No credentials found for given 'iss'"}` when trying to access /get endpoint with the JWT, i'm sure that iss field in the payload is matching the key field in the secret. also, logs seem clean. any ideas what could be the reason? 

Thank you

**Comment by rainest** [comment]
Hey @ahmed-adly-khalil, thanks for the report. I've opened #927 to change this, as I noticed that we still used the old annotation in several other locations also, and wanted to clear all of those at once.

If you see issues like this in the future, and know what needs to change, we appreciate PRs from the community, and documentation fixes are a great way to get started there, since they don't require as much local testing infrastructure as code changes. https://github.com/Kong/kubernetes-ingress-controller/blob/main/CONTRIBUTING.md has information on what you'll need if you are interested in contributing code as well.
